ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -53,6 +53,47 @@ tests:
   container:
     clone: true
     from: golangci-lint
+- as: check-commits-count
+  commands: ci/prow/check-commits-count
+  container:
+    clone: true
+    from: src
+  optional: true
+- as: e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 40m0s
+    version: "4.11"
+  steps:
+    test:
+    - as: e2e-incluster-build
```

Comment:
Let's remove the `e2e` prefix from the steps names. CI is already prefixing the job name:
```
Running step e2e-e2e-incluster-build
```